corehq/apps/sms/views.py

```diff
@@ -160,6 +160,7 @@ def dispatch(self, *args, **kwargs):
         return super(BaseMessagingSectionView, self).dispatch(*args, **kwargs)
 
 
+@csrf_exempt
 def post(request, domain):
     """
     I don't know of anywhere this is being invoked from. If the soft asserts
```

Comment:
i'm still pretty sure nothing is calling this, but i realized i forgot to make it csrf exempt so i might be missing any soft asserts from post requests (i haven't gotten any soft asserts from gets).  i figured i'd leave this up there for another week or so and then remove it entirely if i don't get any soft asserts from the posts.